providers/domain.rb

```diff
@@ -82,7 +81,7 @@
 end
 
 action :join do
-  if exists?
+  if !exists?
```

Comment:
In Ruby it is more idiomatic to use `unless` instead of`ifi !`
Most style checkers would signal this as a violation to the language convention.